components/ResearchingResilience.js

```diff
@@ -0,0 +1,28 @@
+// @author Jimmy Yang
+import React from 'react'
+import Title from './Title'
+import Button from './Button'
+
+export default function ResearchingResilience() {
```

Comment:
good component name!

Reply:
Actually, do you mind switching it do just function ResearchingResilience() {...} and then export default ResearchingResilience at the end? That way we stay consistent throughout the project. Looks good otherwise :3